app/presenters/vendor_api/single_application_presenter.rb

```diff
@@ -34,6 +34,7 @@ def application_as_json
       {
         id: application_choice.id.to_s,
         type: 'application',
+        url: provider_interface_application_choice_url(application_choice),
```

Comment:
maybe `application_url`? I was looking at the safeguarding url and maybe this makes them aligned 🤔